script.js

```diff
@@ -55,27 +58,29 @@ const title = [
 
 const plchdr = [
   {
-    cls: "img",
+    cls: "plchdr",
+    cls1: "img",
     src: "../images/plc-img.jpg",
     alt: "A woman doing yoga",
-    cls1: "text",
+    cls2: "text",
     h2: "Multi-Post-Stories",
     para: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
     list: ["css", "html", "bootstrap", "ruby"],
     btn: "See Project",
+
+    cls3: "cProject",
   },
 ];
 

```

Comment:
- [ ] @team kindly ensure your projects array is in the format as shown below,
![image](https://user-images.githubusercontent.com/61408860/202845084-e630c938-051a-4c68-9bc2-c00aaccb12de.png)
 this will make it easier for you to just loop through the array and add the classes in just one loop instead of how you have implemented it. 
This is a project requirement:
![image](https://user-images.githubusercontent.com/61408860/202845098-348a4175-1f4a-476d-8434-f15a992b77fa.png)